.gitignore

```diff
@@ -12,3 +11,4 @@ docs/
 npm-debug.log
 /.idea/
 .env
+dist/*
```

Comment:
I see. I guess we can do this with [`git add --force`](https://git-scm.com/docs/git-add). I am 👍  then. Sounds like a good idea.

So after this we will use only `dist` for built files, but we will only commit the built files of each release by doing it verbosely. Sounds even better.

Am I right here?